### PR TITLE
Assorted fixes

### PR DESCRIPTION
**Pull request by blimpa**
- Language fixes
- Removed redundantewrong component (I made sure there were no dangling references to any of the components whose numbers were changed)

### CODE REVIEW

File: MIP101/MIP101.md [hunk 1]
```diff
@@ -413,7 +413,7 @@ There must always be clear justification for all Alignment Artifact strengthenin
 
 ###### 2.5.1.2.2
 
-The Alignment Artifact strengthening work of AVCs must properly take into account and justify its impact on Future Alignment Risk of the AVC.
```
Comment (Le-Bateleur-2023) [inline MIP101/MIP101.md]: Let's leave this as is. The Author intends to emphasize this phrase.

File: MIP113/MIP113.md [hunk 2]
```diff
@@ -160,7 +160,7 @@ The Advisory Council is paid on a project basis to do specific work that improve
...
+Entities are encouraged to submit applications with the intention of enhancing a specific portion of the Scope Framework. Any entity notice a problem in the Scope or something that can and should be improved and apply to undergo a process to suggest improvements. The Governance Facilitators should diligently encourage participants possessing relevant areas of expertise to actively engage in this process to the fullest extent possible. Accordingly, once this MIP is successfully passed, it is imperative that the Governance Facilitators promptly post a Request for Applicants to the forum within a period of 30 days. The Governance Facilitators may choose to utilize a Peer-to-Peer recruiting process, which involves allocating a portion of the Governance Advisory Council budget to incentivize ecosystem participants and community members to actively seek out applicants, providing comprehensive explanations of the process.
```
Comment (Le-Bateleur-2023) [inline MIP113/MIP113.md]:
```suggestion
Entities are encouraged to submit applications with the intention of enhancing a specific portion of the Scope Framework. Any entity that notices a problem in the Scope or something that can and should be improved can apply to undergo a process to suggest improvements. The Governance Facilitators should diligently encourage participants possessing relevant areas of expertise to actively engage in this process to the fullest extent possible. Accordingly, once this MIP is successfully passed, it is imperative that the Governance Facilitators promptly post a Request for Applicants to the forum within a period of 30 days. The Governance Facilitators may choose to utilize a Peer-to-Peer recruiting process, which involves allocating a portion of the Governance Advisory Council budget to incentivize ecosystem participants and community members to actively seek out applicants, providing comprehensive explanations of the process.
```

**Review by Le-Bateleur-2023 — COMMENTED** [review]
Thanks @blimpa ! 

I made some minor changes. When you have finalized the PR, ping me and please add VoteWizard and JanSky as reviewers, bc we need consensus from Facilitators to directly edit Scopes (per GOV3).

I'll also ping them in Discord to notify them.

**Review by Le-Bateleur-2023 — APPROVED** [review]
looks good!

**Comment by Le-Bateleur-2023** [comment]
@MakerJanSky and @VoteWizard 

minor fixes have been proposed to this MIP. they are of the nature of clear typos and thus per GOV3, a consensus of Facilitators can directly edit to correct the problems. 

Please review and let me know if you both agree to approve and merge the GOV3 edits.

File: MIP101/MIP101.md (hunk 1 above)
Comment (MakerJanSky) [reply MIP101/MIP101.md]: Bumping this to prevent accidental merging.

**Review by MakerJanSky — CHANGES_REQUESTED** [review]

File: MIP101/MIP101.md (hunk 1 above)
Comment (Le-Bateleur-2023) [reply MIP101/MIP101.md]: @blimpa please reverse this edit; it should not be part of the PR.

**Review by Le-Bateleur-2023 — APPROVED** [review]
looks good!

**Pull request merged**